bok_choy/browser.py

```diff
@@ -159,13 +161,14 @@ def browser(tags=None, proxy=None):
     1. Local browsers: If the proper environment variables are not all set for the second case,
         then we use a local browser.
 
-        * The environment variable `SELENIUM_BROWSER` can be set to
-        specify which local browser to use. The default is Firefox.
-        * Additionally, if a proxy
-        instance is passed and the browser choice is either Chrome or Firefox, then the browser will
-        be initialized with the proxy server set.
-        * The environment variable `SELENIUM_FIREFOX_PATH` can be used for specifying a path to the
-        Firefox binary. Default behavior is to use the system location.
+        * The environment variable `SELENIUM_BROWSER` can be set to specify which local browser to use. The default is \
```

Comment:
I added those because the rendered HTML was treating the newlines in the docstring as the end of the list element and that was breaking the list formatting (see http://bok-choy.readthedocs.io/en/latest/api_reference.html#bok_choy.browser.browser)